.env.template

```diff
@@ -52,4 +52,6 @@ AUTHORIZED_USERS='{"admin":"SECRET","accountant":"SECRET"}'
 DEFAULT_FINANCIAL_ENTITY_ID=00000000-0000-0000-0000-000000000000
 
 ; hive
-HIVE_TOKEN=hiveToken
\ No newline at end of file
+HIVE_REGISTRY_TOKEN=hiveRegistryToken
```

Comment:
why do we have a placeholder here? It can be just an empty string. 
Also, I think we can avoid this if the composition happens locally during development?